ntbs-service/Pages/Notifications/OverviewPartials/_ContactTracingOverviewPartial.cshtml

```diff
@@ -17,9 +17,9 @@
                     <div> Total  </div>
                 </div>
                 <div>
-                    <div class="contact-tracing-value"> @Model.Notification.ContactTracing.AdultsIdentified </div>
-                    <div class="contact-tracing-value"> @Model.Notification.ContactTracing.ChildrenIdentified </div>
-                    <div class="contact-tracing-value"> @Model.Notification.TotalContactsIdentified </div>
+                    <div class="contact-tracing-value"> @(Model.Notification.ContactTracing.AdultsIdentified ?? Html.Raw(emptySpace))</div>

```

Comment:
are we not better off by putting a minimal width on the `contact-tracing-value` class?

Reply:
Wait, that's not right - our markup is the wrong way round! I think using a table here would be the cleanest approach actually..